diff -Naur blender-2.77a.old/blender.thumbnailer blender-2.77a/blender.thumbnailer
--- blender-2.77a.old/blender.thumbnailer	1970-01-01 01:00:00.000000000 +0100
+++ blender-2.77a/blender.thumbnailer	2016-06-12 14:17:06.018593810 +0200
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff -Naur blender-2.77a.old/source/creator/CMakeLists.txt blender-2.77a/source/creator/CMakeLists.txt
--- blender-2.77a.old/source/creator/CMakeLists.txt	2016-06-12 14:16:59.136458133 +0200
+++ blender-2.77a/source/creator/CMakeLists.txt	2016-06-12 14:17:06.020593849 +0200
@@ -497,7 +497,7 @@
 		)
 		install(
 			PROGRAMS ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION bin
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
 		)
 		set(BLENDER_TEXT_FILES_DESTINATION share/doc/blender)
 	endif()
