--- blender-2.78a/extern/glog/src/stacktrace_powerpc-inl.h.debug	2016-10-24 23:13:56.000000000 +0900
+++ blender-2.78a/extern/glog/src/stacktrace_powerpc-inl.h	2016-11-12 14:55:59.832015717 +0900
@@ -111,10 +111,10 @@
       result[n++] = *(sp+2);
 #elif defined(_CALL_SYSV)
       result[n++] = *(sp+1);
-#elif defined(__APPLE__) || (defined(__linux) && defined(__PPC64__))
+#elif defined(__APPLE__) || (defined(__linux__) && defined(__PPC64__))
       // This check is in case the compiler doesn't define _CALL_AIX/etc.
       result[n++] = *(sp+2);
-#elif defined(__linux)
+#elif defined(__linux__)
       // This check is in case the compiler doesn't define _CALL_SYSV.
       result[n++] = *(sp+1);
 #else
