--- blender-2.80.old/source/blender/blenfont/intern/blf_font_i18n.c	2019-07-24 00:41:39.000000000 -0700
+++ blender-2.80/source/blender/blenfont/intern/blf_font_i18n.c	2019-07-31 19:55:46.069808055 -0700
@@ -50,7 +50,7 @@
 
 static void fontbuf_load(struct FontBuf *fb)
 {
-  const char *fontpath = BKE_appdir_folder_id(BLENDER_DATAFILES, "fonts");
+  const char *fontpath = "/usr/share/fonts/blender";
   if (fontpath) {
     char unifont_path[1024];
     BLI_snprintf(unifont_path, sizeof(unifont_path), "%s/%s", fontpath, fb->filename);
--- blender-2.80.old/source/creator/CMakeLists.txt	2019-07-24 00:41:39.000000000 -0700
+++ blender-2.80/source/creator/CMakeLists.txt	2019-08-05 17:38:34.223948608 -0700
@@ -393,9 +393,8 @@
 # localization
 if(WITH_INTERNATIONAL)
   install(
-    DIRECTORY
-      ${CMAKE_SOURCE_DIR}/release/datafiles/fonts
-    DESTINATION ${TARGETDIR_VER}/datafiles
+    DIRECTORY ${CMAKE_SOURCE_DIR}/release/datafiles/fonts/
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/fonts/blender
   )
 
   set(_locale_dir "${CMAKE_SOURCE_DIR}/release/datafiles/locale")
