diff -Naur blender-2.78a.old/intern/ghost/intern/GHOST_SystemPathsUnix.cpp blender-2.78a/intern/ghost/intern/GHOST_SystemPathsUnix.cpp
--- blender-2.78a.old/intern/ghost/intern/GHOST_SystemPathsUnix.cpp	2017-01-29 23:04:06.462691400 +0100
+++ blender-2.78a/intern/ghost/intern/GHOST_SystemPathsUnix.cpp	2017-01-29 23:12:27.057495546 +0100
@@ -61,6 +61,9 @@
 
 const GHOST_TUns8 *GHOST_SystemPathsUnix::getSystemDir(int, const char *versionstr) const
 {
+	/* ignore versionstr when building the system path */
+	versionstr = "";
+
 	/* no prefix assumes a portable build which only uses bundled scripts */
 	if (static_path) {
 		static string system_path = string(static_path) + "/blender/" + versionstr;
diff -Naur blender-2.78a.old/source/creator/CMakeLists.txt blender-2.78a/source/creator/CMakeLists.txt
--- blender-2.78a.old/source/creator/CMakeLists.txt	2017-01-29 23:12:09.267182657 +0100
+++ blender-2.78a/source/creator/CMakeLists.txt	2017-01-29 23:12:27.058495563 +0100
@@ -306,7 +306,7 @@
 		if(WITH_INSTALL_PORTABLE)
 			set(TARGETDIR_VER ${BLENDER_VERSION})
 		else()
-			set(TARGETDIR_VER share/blender/${BLENDER_VERSION})
+			set(TARGETDIR_VER share/blender)
 		endif()
 	endif()
 
