diff -Naur blender-2.77a.old/source/creator/CMakeLists.txt blender-2.77a/source/creator/CMakeLists.txt
--- blender-2.77a.old/source/creator/CMakeLists.txt	2016-06-12 14:17:51.988500075 +0200
+++ blender-2.77a/source/creator/CMakeLists.txt	2016-06-12 14:17:34.259150553 +0200
@@ -327,8 +327,8 @@
 	endif()
 
 	install(
-		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts
-		DESTINATION ${TARGETDIR_VER}
+		DIRECTORY ${CMAKE_SOURCE_DIR}/release/scripts/
+		DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts/
 		PATTERN ".git" EXCLUDE
 		PATTERN "__pycache__" EXCLUDE
 		PATTERN "${ADDON_EXCLUDE_CONDITIONAL}" EXCLUDE
