diff -Naur blender-2.78a.old/blender.thumbnailer blender-2.78a/blender.thumbnailer
--- blender-2.78a.old/blender.thumbnailer	1970-01-01 01:00:00.000000000 +0100
+++ blender-2.78a/blender.thumbnailer	2017-01-29 23:04:47.528413638 +0100
@@ -0,0 +1,4 @@
+[Thumbnailer Entry]
+TryExec=/usr/share/blender/scripts/blender-thumbnailer.py
+Exec=/usr/share/blender/scripts/blender-thumbnailer.py %i %o
+MimeType=application/x-blender;
diff -Naur blender-2.78a.old/source/creator/CMakeLists.txt blender-2.78a/source/creator/CMakeLists.txt
--- blender-2.78a.old/source/creator/CMakeLists.txt	2017-01-29 23:04:25.162020270 +0100
+++ blender-2.78a/source/creator/CMakeLists.txt	2017-01-29 23:04:47.529413656 +0100
@@ -526,7 +526,7 @@
 		)
 		install(
 			PROGRAMS ${CMAKE_SOURCE_DIR}/release/bin/blender-thumbnailer.py
-			DESTINATION bin
+			DESTINATION ${CMAKE_INSTALL_PREFIX}/share/blender/scripts
 		)
 		set(BLENDER_TEXT_FILES_DESTINATION share/doc/blender)
 	endif()
