--- blender-2.81.old/intern/numaapi/source/build_config.h	2019-10-21 07:04:53.000000000 -0700
+++ blender-2.81/intern/numaapi/source/build_config.h	2019-11-21 20:14:25.629780156 -0800
@@ -318,6 +318,16 @@
 #  else
 #    error Please define your endianness
 #  endif
+#elif (defined(__PPC64__) || defined(__PPC__)) && defined(__BIG_ENDIAN__)
+#  define ARCH_CPU_PPC64_FAMILY 1
+#  define ARCH_CPU_PPC64 1
+#  define ARCH_CPU_64_BITS 1
+#  define ARCH_CPU_BIG_ENDIAN 1
+#elif defined(__PPC64__)
+#  define ARCH_CPU_PPC64_FAMILY 1
+#  define ARCH_CPU_PPC64 1
+#  define ARCH_CPU_64_BITS 1
+#  define ARCH_CPU_LITTLE_ENDIAN 1
 #elif defined(__s390x__)
 #  define ARCH_CPU_S390_FAMILY 1
 #  define ARCH_CPU_S390X 1
@@ -361,6 +371,9 @@
 #if !defined(ARCH_CPU_PPC_FAMILY)
 #  define ARCH_CPU_PPC_FAMILY 0
 #endif
+#if !defined(ARCH_CPU_PPC64_FAMILY)
+#  define ARCH_CPU_PPC64_FAMILY 0
+#endif
 #if !defined(ARCH_CPU_S390_FAMILY)
 #  define ARCH_CPU_S390_FAMILY 0
 #endif
