diff -ru blender-2.78b.orig/source/blender/gpu/intern/gpu_extensions.c blender-2.78b/source/blender/gpu/intern/gpu_extensions.c
--- blender-2.78b.orig/source/blender/gpu/intern/gpu_extensions.c	2017-02-09 01:49:32.000000000 -0800
+++ blender-2.78b/source/blender/gpu/intern/gpu_extensions.c	2017-02-24 23:28:45.158724978 -0800
@@ -173,7 +173,10 @@
 		GG.device = GPU_DEVICE_INTEL;
 		GG.driver = GPU_DRIVER_OFFICIAL;
 	}
-	else if (strstr(renderer, "Mesa DRI R") || (strstr(renderer, "Gallium ") && strstr(renderer, " on ATI "))) {
+	else if ((strstr(renderer, "Mesa DRI R")) ||
+             (strstr(renderer, "Gallium ") && strstr(renderer, " on ATI ")) ||
+             (strstr(renderer, "Gallium ") && strstr(renderer, " on AMD ")))
+   {
 		GG.device = GPU_DEVICE_ATI;
 		GG.driver = GPU_DRIVER_OPENSOURCE;
 	}
