diff -Naur blender-2.73a.old/extern/cuew/auto/cuew_gen.py blender-2.73a/extern/cuew/auto/cuew_gen.py
--- blender-2.73a.old/extern/cuew/auto/cuew_gen.py	2015-04-03 10:08:25.064206612 +0200
+++ blender-2.73a/extern/cuew/auto/cuew_gen.py	2015-04-03 10:09:00.698950189 +0200
@@ -446,7 +446,7 @@
   /* Default installation path. */
   const char *path = "/usr/local/cuda/lib/libcuda.dylib";
 #else
-  const char *path = "libcuda.so";
+  const char *path = "libcuda.so.1";
 #endif""")
 
 
diff -Naur blender-2.73a.old/extern/cuew/src/cuew.c blender-2.73a/extern/cuew/src/cuew.c
--- blender-2.73a.old/extern/cuew/src/cuew.c	2015-04-03 10:08:25.064206612 +0200
+++ blender-2.73a/extern/cuew/src/cuew.c	2015-04-03 10:09:00.695950209 +0200
@@ -277,7 +277,7 @@
   /* Default installation path. */
   const char *path = "/usr/local/cuda/lib/libcuda.dylib";
 #else
-  const char *path = "libcuda.so";
+  const char *path = "libcuda.so.1";
 #endif
   static int initialized = 0;
   static int result = 0;
diff -Naur blender-2.73a.old/intern/cycles/kernel/CMakeLists.txt blender-2.73a/intern/cycles/kernel/CMakeLists.txt
--- blender-2.73a.old/intern/cycles/kernel/CMakeLists.txt	2015-04-03 10:08:25.092206410 +0200
+++ blender-2.73a/intern/cycles/kernel/CMakeLists.txt	2015-04-03 10:09:14.230857804 +0200
@@ -195,6 +195,7 @@
 					${cuda_debug_flags}
 					-I${CMAKE_CURRENT_SOURCE_DIR}/../util
 					-I${CMAKE_CURRENT_SOURCE_DIR}/svm
+					-I/usr/include/cuda
 					-DCCL_NAMESPACE_BEGIN=
 					-DCCL_NAMESPACE_END=
 					-DNVCC
