diff -Naur blender-2.78.old/extern/cuew/src/cuew.c blender-2.78/extern/cuew/src/cuew.c
--- blender-2.78.old/extern/cuew/src/cuew.c	2016-10-14 19:42:34.178431536 +0200
+++ blender-2.78/extern/cuew/src/cuew.c	2016-10-14 19:48:07.380159055 +0200
@@ -323,7 +323,7 @@
   const char *cuda_paths[] = {"/usr/local/cuda/lib/libcuda.dylib", NULL};
   const char *nvrtc_paths[] = {"/usr/local/cuda/lib/libnvrtc.dylib", NULL};
 #else
-  const char *cuda_paths[] = {"libcuda.so", NULL};
+  const char *cuda_paths[] = {"libcuda.so.1", NULL};
   const char *nvrtc_paths[] = {"libnvrtc.so",
 #  if defined(__x86_64__) || defined(_M_X64)
                                "/usr/local/cuda/lib64/libnvrtc.so",
diff -Naur blender-2.78.old/intern/cycles/kernel/CMakeLists.txt blender-2.78/intern/cycles/kernel/CMakeLists.txt
--- blender-2.78.old/intern/cycles/kernel/CMakeLists.txt	2016-10-14 19:42:33.912426927 +0200
+++ blender-2.78/intern/cycles/kernel/CMakeLists.txt	2016-10-14 19:48:34.594625892 +0200
@@ -274,6 +274,9 @@
 					${cuda_debug_flags}
 					-I${CMAKE_CURRENT_SOURCE_DIR}/../util
 					-I${CMAKE_CURRENT_SOURCE_DIR}/svm
+					-I/usr/include/cuda
+					-std=c++11
+					-D__CORRECT_ISO_CPP11_MATH_H_PROTO
 					-DCCL_NAMESPACE_BEGIN=
 					-DCCL_NAMESPACE_END=
 					-DNVCC
