diff -Naur blender-2.79.old/intern/cycles/kernel/CMakeLists.txt blender-2.79/intern/cycles/kernel/CMakeLists.txt
--- blender-2.79.old/intern/cycles/kernel/CMakeLists.txt	2017-09-21 16:24:23.457885865 +0200
+++ blender-2.79/intern/cycles/kernel/CMakeLists.txt	2017-09-21 16:24:55.087470048 +0200
@@ -372,6 +372,8 @@
 			COMMAND ${cuda_nvcc_command}
 					-arch=${arch}
 					${CUDA_NVCC_FLAGS}
+					-I/usr/include/cuda
+					-ccbin /usr/bin/cuda-g++
 					-m${CUDA_BITS}
 					--cubin ${CMAKE_CURRENT_SOURCE_DIR}${cuda_kernel_src}
 					-o ${CMAKE_CURRENT_BINARY_DIR}/${cuda_cubin}
